lms/djangoapps/discussion_api/serializers.py

```diff
@@ -198,10 +200,19 @@ def get_non_endorsed_comment_list_url(self, obj):
         """Returns the URL to retrieve the thread's non-endorsed comments."""
         return self.get_comment_list_url(obj, endorsed=False)
 
+    def validate_course_id(self, attrs, _source):
```

Comment:
i probably missed this in an earlier PR, but why isn't course_id a read-only field?